Documentation/specs/runtime-configuration-file.md

```diff
@@ -10,21 +10,25 @@ There are two runtime configuration files for a particular application. Given a
 
 * `MyApp.dll` - The managed assembly for `MyApp`, including an ECMA-compliant entry point token.
 * `MyApp.exe` - A copy of the `corehost.exe` executable.
-* `MyApp.config.json` - An **optional** configuration file containing runtime configuration settings.
+* `MyApp.runtimeconfig.json` - An **optional** configuration file containing runtime configuration settings.
 * `MyApp.deps.json` - A list of dependencies, as well as compilation context data and compilation dependencies. Not technically required, but required to use the servicing or package cache/shared package install features.
 
 **IMPORTANT**: Portable Applications, i.e. those published without a specific RID, have some adjustments to this spec which is covered at the end.
 
 ## File format
 
-The files are both JSON files stored in UTF-8 encoding. Below are sample files. Note that not all sections are required and some will be opt-in only (see below for more details). The `.config.json` file is completely optional, and in the `.deps.json` file, only the `runtimeTarget`, `targets` and `libraries` sections are required (and within the `targets` section, only the runtime-specific target is required).
+The files are both JSON files stored in UTF-8 encoding. Below are sample files. Note that not all sections are required and some will be opt-in only (see below for more details). The `.runtimeconfig.json` file is completely optional, and in the `.deps.json` file, only the `runtimeTarget`, `targets` and `libraries` sections are required (and within the `targets` section, only the runtime-specific target is required).
 
-### [appname].config.json
+### [appname].runtimeconfig.json
 ```json
 {
-    "runtimeConfig": {
+    "runtimeOptions": {
         "gcServer": true,
         "gcConcurrent": false
+    },
+    "targetRuntime": {
```

Comment:
No, `runtimeTarget` is expressing which entry in the `targets` section `corehost` should use, since we don't want `corehost` to deal with parsing Target Names.